package.json

```diff
@@ -1,135 +1,22 @@
-  },
-  "peerDependenciesMeta": {
-    "@ember/test-helpers": {
-      "optional": true
-    }
+    "build": "pnpm --filter ember-scroll-modifiers run build",
+    "lint": "pnpm --filter ember-scroll-modifiers run lint",
+    "lint:fix": "pnpm --filter ember-scroll-modifiers run lint:fix",
+    "prepare": "pnpm build",
+    "start": "concurrently 'pnpm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "pnpm --filter ember-scroll-modifiers ember-scroll-modifiers run start",
+    "start:test-app": "pnpm --filter ember-scroll-modifiers test-app run start",
+    "test": "pnpm --filter ember-scroll-modifiers run test"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.25.1",
-    "@babel/plugin-proposal-decorators": "^7.24.7",
-    "@babel/plugin-transform-class-static-block": "^7.24.7",
-    "@ember/optional-features": "^2.1.0",
-    "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^4.0.2",
-    "@embroider/test-setup": "^4.0.0",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.2.2",
-    "ember-cli": "~5.11.0",
-    "ember-cli-clean-css": "^3.0.0",
-    "ember-cli-dependency-checker": "^3.3.2",
-    "ember-cli-deprecation-workflow": "^2.0.0",
-    "ember-cli-fastboot": "^4.1.5",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sass": "^11.0.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-data": "~5.3.8",
-    "ember-fetch": "^8.1.2",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^8.2.3",
-    "ember-qunit": "^8.1.0",
-    "ember-resolver": "^11.0.1",
-    "ember-sinon-qunit": "^6.0.0",
-    "ember-source": "~5.11.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^6.0.0",
-    "ember-try": "^3.0.0",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-ember": "^12.2.0",
-    "eslint-plugin-n": "^17.10.2",
-    "eslint-plugin-prettier": "^5.2.1",
-    "eslint-plugin-qunit": "^8.1.2",
-    "field-guide": "^2.4.0",
-    "field-guide-default-template": "^3.0.0",
-    "loader.js": "^4.7.0",
-    "prember": "^2.1.0",
-    "prettier": "^3.3.3",
-    "qunit": "^2.22.0",
-    "qunit-dom": "^3.2.0",
-    "release-it": "^15.11.0",
-    "release-it-lerna-changelog": "^5.0.0",
-    "sass": "^1.77.8",
-    "stylelint": "^15.11.0",
-    "stylelint-config-standard": "^34.0.0",
-    "stylelint-prettier": "^4.1.0",
-    "webpack": "^5.94.0"
-  },
-  "fastbootDependencies": [
-    "crypto"
-  ],
-  "engines": {
-    "node": "16.* || >= 18"
-  },
-  "volta": {
-    "node": "18.20.4",
-    "yarn": "1.22.22"
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config",
-    "demoURL": "https://ember-scroll-modifiers.jhawk.co/"
+    "concurrently": "^8.2.2"
   },
-  "homepage": "https://ember-scroll-modifiers.jhawk.co/",
-  "release-it": {
```

Comment:
We'll need to keep a version of this: https://github.com/elwayman02/ember-sinon-qunit/blob/master/package.json#L38

Reply:
We'll need to add the workspace support for it as well: https://github.com/elwayman02/ember-sinon-qunit/blob/master/package.json#L34